addons/medical_vitals/functions/fnc_updateOxygen.sqf

```diff
@@ -0,0 +1,73 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Brett Mayson
+ * Update the oxygen levels
+ *
+ * Arguments:
+ * 0: The Unit <OBJECT>
+ * 1: Time since last update <NUMBER>
+ * 2: Sync value? <BOOL>
+ *
+ * ReturnValue:
+ * Current SPO2 <NUMBER>
+ *
+ * Example:
+ * [player, 1, false] call ace_medical_vitals_fnc_updateOxygen
+ *
+ * Public: No
+ */
+
+params ["_unit", "_deltaT", "_syncValue"];
+
+if (!GVAR(simulateSpO2)) exitWith {};
```

Comment:
If someone decides to turn off SpO2 during the mission, the affected values won't regenerate, which I assume could be problematic.

Imo there are two solutions:
- Make the SpO2 simulation setting require a restart when changed.
- Update the variables to their default values when SpO2 simulation is disabled.

I prefer the second option personally, because it allows for more flexibility.

Reply:
I imagine this should do the trick:

```suggestion
if (!GVAR(simulateSpO2)) exitWith {
    _unit setVariable [QGVAR(oxygenDemand), 0];
    _unit setVariable [VAR_SPO2, DEFAULT_SPO2, _syncValue];
};
```